### PR TITLE
DOCS: Small fixes per docs feedback project

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**

### CODE REVIEW

File: gpdb-doc/dita/install_guide/init_gpdb.xml [hunk 1]
```diff
@@ -184,17 +184,21 @@ declare -a MIRROR_DATA_DIRECTORY=(/data1/mirror /data1/mirror /data1/mirror /dat
...
                     output_configuration_file</varname></codeph> option to save the cluster
                 configuration details in a file. For
                 example:</p><codeblock>$ gpinitsystem -c gpconfigs/gpinitsystem_config -O gpconfigs/config_template </codeblock><p>This
                 output file can be edited and used at a later stage as the input file of the
                   <codeph>-I</codeph> option, to create a new cluster or to recover from a backup.
                 See <xref href="../utility_guide/ref/gpinitsystem.xml" type="topic" scope="peer"
-                  >gpinitsystem</xref> for further details.</p></li>
+                  >gpinitsystem</xref> for further details. <note>Calling
+                    <codeph>gpinitsystem</codeph> with the <codeph>-O</codeph> option does not
+                  initialize the Greenplum Database system; it merely saves a file with cluster
```
Comment (lisakowen) [inline gpdb-doc/dita/install_guide/init_gpdb.xml:200]: saves -> generates and saves?

File: gpdb-doc/dita/install_guide/init_gpdb.xml [hunk 2]
```diff
@@ -184,17 +184,21 @@ declare -a MIRROR_DATA_DIRECTORY=(/data1/mirror /data1/mirror /data1/mirror /dat
...
+  -s <varname>standby_master_hostname</varname> --mirror-mode=spread</codeblock><p>During
                 a new cluster creation, you may use the <codeph><varname>-O
                     output_configuration_file</varname></codeph> option to save the cluster
                 configuration details in a file. For
                 example:</p><codeblock>$ gpinitsystem -c gpconfigs/gpinitsystem_config -O gpconfigs/config_template </codeblock><p>This
                 output file can be edited and used at a later stage as the input file of the
                   <codeph>-I</codeph> option, to create a new cluster or to recover from a backup.
                 See <xref href="../utility_guide/ref/gpinitsystem.xml" type="topic" scope="peer"
-                  >gpinitsystem</xref> for further details.</p></li>
+                  >gpinitsystem</xref> for further details. <note>Calling
```
Comment (lisakowen) [inline gpdb-doc/dita/install_guide/init_gpdb.xml:198]: wondering if we should also add this note to the gpinitsystem ref page?

**Review by lisakowen — APPROVED** [review]
looks good, a few minor comments.

File: gpdb-doc/dita/install_guide/init_gpdb.xml (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/dita/install_guide/init_gpdb.xml:200]: Good catch.

File: gpdb-doc/dita/install_guide/init_gpdb.xml (hunk 2 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/dita/install_guide/init_gpdb.xml:198]: Yes.

**Review by dyozie — APPROVED** [review]

**Pull request merged**